dev/protos/firestore_proto_api.d.ts

```diff
@@ -3176,7 +3176,7 @@ export namespace google {
 
                     /** Operator enum. */
                     type Operator =
-                        "OPERATOR_UNSPECIFIED"| "LESS_THAN"| "LESS_THAN_OR_EQUAL"| "GREATER_THAN"| "GREATER_THAN_OR_EQUAL"| "EQUAL"| "ARRAY_CONTAINS";
+                        "OPERATOR_UNSPECIFIED"| "LESS_THAN"| "LESS_THAN_OR_EQUAL"| "GREATER_THAN"| "GREATER_THAN_OR_EQUAL"| "EQUAL"| "ARRAY_CONTAINS" | "IN" | "ARRAY_CONTAINS_ANY";
```

Comment:
Before you merge this you should run https://github.com/googleapis/nodejs-firestore/blob/master/dev/protos/update.sh. Ideally, this would not produce a diff (once the public Protos are available).